grammar/Graql.g4

```diff
@@ -355,3 +355,4 @@ fragment ESCAPE_SEQ_    : '\\' . ;
 
 COMMENT         : '#' .*? '\r'? ('\n' | EOF)    -> channel(HIDDEN) ;
 WS              : [ \t\r\n]+                    -> channel(HIDDEN) ;
+ErrorCharacter : . ;
```

Comment:
Can we look into why this fixes the issue, and if there are any side effects to the Graql language, @BFergerson @flyingsilverfin ? And if it's the correct use, please update the PR description, as well as the alignment of the colon `:` to match the lines above it.

Reply:
Fixed the alignment. I'm unsure what you mean about updating PR description. The PR description already mentions this change and why it was done.

Reply:
I'll investigate why we have to have the `ErrorCharacter` and let you know :)
Sorry to be so picky on this, we're very protective of the Graql repository since it's one of the main value props of grakn! We'll get this merged soon!

Reply:
No rush on my part. I'm already using this plugin :).

This is what I was running into: https://github.com/antlr/antlr4-intellij-adaptor/wiki/Getting-started#about-grammars

## Recreating
Make schema
```
define
    person sub entity;
```

Modify to:
```
define
    person sub entity;
```

Try to delete `@`.

Reply:
The content on the linked page above doesn't say anything else other than not to `-> skip`, @BFergerson - which we're not doing. It doesn't say why we need `ErrorCharacter: . ;` though. Did we ever find out why we need this character?

Reply:
@haikalpribadi, given the schema: `define person@ sub entity;` what should the grammar file recognize `@` as?
I'm no expert on this stuff but I see it parsed as:
![image](https://user-images.githubusercontent.com/3278877/90315793-3f7f8480-df2f-11ea-9e86-1816a04d7c98.png)

Alternatively, when I add `ErrorCharacter: . ;` it is parsed as:
![image](https://user-images.githubusercontent.com/3278877/90315810-5920cc00-df2f-11ea-9ff7-e8107265fbf9.png)

The IntelliJ [documentation](https://jetbrains.org/intellij/sdk/docs/reference_guide/custom_language_support/implementing_lexer.html) says:
>  Lexers, and in particular JFlex-based lexers, need to be created in such a way that they always match the entire contents of the file, without any gaps between tokens, and generate special tokens for characters which are not valid at their location. Lexers must never abort prematurely because of an invalid character.

I don't know how much that applies but from my understanding the issue stems from the fact that the grammar without `ErrorCharacter` fails to recognize the `@` whereas adding the `ErrorCharacter` adds a catchall for anything that the grammar doesn't pick up, making sure every character is matched even when they're erroneous.

As the ANTLR plugin creator suggests, I copied the catchall from here: https://github.com/tunnelvisionlabs/antlr4-grammar-postgresql/blob/4c4d8e6a8f3057b319365ec947e6e76cc5a22e14/src/com/tunnelvisionlabs/postgresql/PostgreSqlLexer.g4#L808

Reply:
Thank you for the thorough explanation, @BFergeson! I completely understand what it's for now, and it makes complete sense! In fact, I should add this now in our current Grammar (as `UNRECOGNISED : . ;`) so it can be in sooner. You can remove it from your PR. Thanks for highlighting this, Brandon!

Reply:
I've merged my commit from `master` actually, so you don't need to make any changes for this. I'll mark this issue resolved now.